openslides_backend/action/base.py

```diff
@@ -208,7 +208,7 @@ def get_structured_fields_in_instance(
                 r"^"
                 + field_name[: field.index]
                 + r"\$"
-                + r"([^_]*)"
+                + r"(.*)"
```

Comment:
Prefix has to be unique among all structured fields, so we can match everything here. Especially important to match replacements with underscores. @FinnStutzenstein we should check in the validator of the models file that the prefixes are unique, please remember this when we move it :D

Reply:
Ok, I was working on the validator this morning; I added a note.